### PR TITLE
Fix validation TF

### PR DESCRIPTION
**Pull request by sonchang**

### CODE REVIEW

**Comment by sonchang** [comment]
@cjellick @alena1108

File: cattle/plugins/docker/compute.py [hunk 1]
```diff
@@ -278,6 +278,14 @@ def instance_activate(self, req=None, instanceHostMap=None,
 
         instance = instanceHostMap.instance
         host = instanceHostMap.host
+
+        try:
+            host.clusterConnection = \
+                instanceHostMap.data.fields['clusterConnection']
+            log.debug('clusterConnection = %', host.clusterConnection)
```
Comment (sonchang) [inline cattle/plugins/docker/compute.py]: since it's logged at debug, do you think that's ok?  i can even change it to trace too

Comment (alena1108) [inline cattle/plugins/docker/compute.py]: trace would be better

**Comment by alena1108** [comment]
LGTM

**Comment by cjellick** [comment]
LGTM

**Pull request merged**